pooltasks.md

```diff
@@ -63,6 +63,32 @@ This is a **read-only** endpoint.
 
 It returns the tasks available for the specified user
 
+#### findAllByItem
+**/api/workflow/pooltasks/search/findAllByItem?uuid=<:item-uuid>**
+It returns all the pool tasks related to the specified item
```

Comment:
Same here, I'd prefer we note this is an ADMIN only endpoint in the description as well.